docs/Inputs.md

```diff
@@ -450,30 +450,20 @@ import { NullableBooleanInput } from 'react-admin';
 
 ![NullableBooleanInput](./img/nullable-boolean-input.png)
 
-`<NullableBooleanInput />` doesn't display the empty option by default. If you want to customize its label and display it, you can use the `displayNull` prop.
+The labels of the options can be customized using the `nullLabel`, `falseLabel` and `trueLabel` properties.
```

Comment:
This is a bit confusing: the doc should explain that in order to customize the labels for the entire app, the right way is to override the translation. Custom labels are only necessary if one wants to have distinct labels for one given input.